### PR TITLE
Added validation check when resolving XSD

### PR DESCRIPTION
**Pull request by garethahealy**
This is to force users to move to the v6 XSD, as we can control that where as the old sourceforge is out of our control.

### CODE REVIEW

**Comment by garethahealy** [comment]
@orange-buffalo ; can you review please.

File: core/src/test/java/org/dozer/Version5XSDTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import org.junit.rules.ExpectedException;
+
+public class Version5XSDTest {
+
+    @Rule
+    public ExpectedException testOldXSDNaming = ExpectedException.none();
+
+    @Test
+    public void testOldXSDNaming() {
+        testOldXSDNaming.expectMessage(Matchers.containsString("Found v5 XSD: 'http://dozer.sourceforge.net/schema/beanmapping.xsd'. Expected v6 XSD: 'http://dozermapper.github.io/schema/bean-mapping.xsd'"));
```
Comment (orange-buffalo) [inline core/src/test/java/org/dozer/Version5XSDTest.java]: One question here about styling. This line is quite long, and I also noticed long lines (>120 characters) all over the Dozer code. Is it planned to adopt any line length convention for the project? IMO, lines longer than 120 characters are hard to read, especially on a laptop screen.

File: core/src/main/java/org/dozer/loader/xml/DozerResolver.java [hunk 2]
```diff
@@ -42,12 +43,18 @@
...
+    private static final String VERSION_5_XSD = "http://dozer.sourceforge.net/schema/beanmapping.xsd";
+    private static final String VERSION_6_XSD = "http://dozermapper.github.io/schema/bean-mapping.xsd";
 
     public InputSource resolveEntity(String publicId, String systemId) {
         InputSource source = null;
 
         log.debug("Trying to resolve XML entity with public ID [{}] and system ID [{}]", publicId, systemId);
 
+        if (VERSION_5_XSD.equalsIgnoreCase(systemId)) {
+            throw new MappingException("Found v5 XSD: '" + VERSION_5_XSD + "'. Expected v6 XSD: '" + VERSION_6_XSD + "'");
```
Comment (orange-buffalo) [inline core/src/main/java/org/dozer/loader/xml/DozerResolver.java]: I would suggest to add a more explicit hint for end-user in exception message. For instance, telling them that starting from v6 new schema location is used and all users must upgrade their mapping files.

File: core/src/test/java/org/dozer/Version5XSDTest.java (hunk 1 above)
Comment (garethahealy) [inline core/src/test/java/org/dozer/Version5XSDTest.java]: Yes, end goal is to adopt same checkstyle as Apache Camel
- https://github.com/apache/camel/blob/master/buildingtools/camel-checkstyle.xml

**Comment by garethahealy** [comment]
@orange-buffalo ; updated as per feedback.

File: core/src/test/java/org/dozer/Version5XSDTest.java (hunk 1 above)
Comment (orange-buffalo) [inline core/src/test/java/org/dozer/Version5XSDTest.java]: Wow, didn't know that Camel used 200-characters long lines. Funny that this does not fit into my screen))

**Pull request merged**